### DISCUSSION TITLE
Is Traffic split Plugin Support "cookie" withthin the rule

### DISCUSSION THREAD

**Discussion by Mar1usSo** [apache/apisix#5096] [Q&A] [answered]
Is traffic split Plugin Support "cookie" withthin the rule ?
I hope to split traffic according cookie ,eg :
1. when cookie user=mike,it route to weighted_upstreams name "blue"
2. when cookie user=mike,it route to default upstreams name "green"

**Comment by spacewander** [comment, marked as answer]
You can use Nginx variables like cookie_user

**Reply by Mar1usSo** [reply]
Got it.
Thanks a lot 👍
